AWS/README.md

```diff
@@ -1,2 +1,5 @@
 # Scenarios
 ## Add the scenarios which you want khaja to speak/create video about
+
+### PLease make a video on how to create and remotely operate FREE ec2 Windows 7/10 instances. Creating Windows ec2 instances doesn't seem to be as simple as of linux instances. There are only Windows server options available in AMI options no options for FREE Win 10/7.
```

Comment:
Asim, AWS is about creating windows servers not really windows7/10 instances.

I would look into your server request

cloud is about hosting servers not desktops